src/Resource/Build.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace WyriHaximus\Travis\Resource;
```

Comment:
It should be, one of the new features is that we can use reserved keywords in a lot more places now. Not 100% sure this is one.